Exam-Service/src/main/java/tw/waterball/judgegirl/examservice/domain/usecases/homework/CreateHomeworkUseCase.java

```diff
@@ -0,0 +1,59 @@
+package tw.waterball.judgegirl.examservice.domain.usecases.homework;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import tw.waterball.judgegirl.commons.exceptions.NotFoundException;
+import tw.waterball.judgegirl.entities.Homework;
+import tw.waterball.judgegirl.examservice.domain.repositories.HomeworkRepository;
+import tw.waterball.judgegirl.problemapi.clients.ProblemServiceDriver;
+
+import javax.inject.Named;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author - wally55077@gmail.com
+ */
+@Named
+@AllArgsConstructor
+public class CreateHomeworkUseCase {
+
+    private final ProblemServiceDriver problemServiceDriver;
+
+    private final HomeworkRepository homeworkRepository;
+
+    public void execute(Request request, Presenter presenter) {
+        String name = request.name;
+        String existsProblemIds = request.problemIds.stream()
+                .filter(this::isProblemExists)
+                .map(String::valueOf)
+                .collect(Collectors.joining(","));
+        Homework homework = new Homework(name, existsProblemIds);
```

Comment:
don't need this anymore since a Homework is a domain object which **must** use `List<Integer>` instead of `"1,2,3"`